rollup.config.js

```diff
@@ -55,12 +64,14 @@ const bundle = (input, output, options = {}) => ({
       },
       include: [/^.+\.min\.js$/]
     }),
-
-    license({
-      banner: `
-<%= pkg.name %> <%= pkg.version %>
-Date: <%= moment().format('YYYY-MM-DD') %>`
-    })
+    banner(`/**
+ * @license ${pkg.name} v${pkg.version}
```

Comment:
Дату грохнуть решили?:( Это же must have

Reply:
I skipped the date specially. Why is it must-have? I compared 5 open source projects and did find any date it their banners.

Reply:
one one hand we have this info in repo
on the other in our artefacts this will be lost. but do we need it there?

year in the license should be enough from copyright's pov and I don't see any reason to track files' dates otherwise.